src/components/Proposal.tsx

```diff
@@ -40,27 +40,34 @@ class ArcProposal extends Component<Props, Entity, Data> {
   public static get Entity() {
     return CreateContextFeed(
       this._EntityContext.Consumer,
-      this._LogsContext.Consumer
+      this._LogsContext.Consumer,
+      "Proposal"
```

Comment:
I don't see any changes to the LoadingView where it actually uses this property

Reply:
Here's where we are using the property in loading view: https://github.com/dOrgTech/DAOcomponents/blob/expose-logs/src/runtime/LoadingView.tsx#L33